packages/gasket-cli/lib/scaffold/fetcher.js

```diff
@@ -78,6 +78,7 @@ module.exports = class PackageFetcher {
     this.cwd = opts.cwd || process.cwd();
     this.packageName = opts.packageName;
     this.tmp = new Fetcher(opts);
+    this.npmconfig = opts.npmconfig;
```

Comment:
As in other places, `npmconfig` should be around anymore (see https://github.com/godaddy/gasket/pull/647)